lib/src/modal_page/sliver_wolt_modal_sheet_page.dart

```diff
@@ -215,6 +218,7 @@ class SliverWoltModalSheetPage {
   /// Creates a page to be built within [WoltScrollableModalSheet].
   const SliverWoltModalSheetPage({
     required this.mainContentSlivers,
+    this.mainContentSliversBuilder,
```

Comment:
Can you add an assertion here and also update the Readme and the examples there with the use of `mainContentSliversBuilder`?